theseus_gui/src/components/ui/Instance.vue

```diff
@@ -0,0 +1,218 @@
+<script setup>
+import { RouterLink } from 'vue-router'
+import { Card, LogOutIcon, PlusIcon } from 'omorphia'
+
+const props = defineProps({
+  display: String,
+  instance: Object,
+})
+</script>
+
+<template>
+  <div>
+    <RouterLink v-if="display === 'list'" class="instance-list-item" :to="`${props.instance.id}`">{{
+      props.instance.name
+    }}</RouterLink>
+    <div
+      class="instance-gallery-item"
+      :style="{
+        backgroundImage: `url(${props.instance.img})`,
+      }"
+      v-else-if="display === 'gallery'"
+    >
+      <p>{{ props.instance.version }}</p>
+      <p>{{ props.instance.name }}</p>
+      <div class="cta"><LogOutIcon /></div>
+    </div>
```

Comment:
I think we can scrap this design and stick to the Spotify-like one. Looks better + more readable